tools/pontoon/ui.md

```diff
@@ -62,11 +62,11 @@ When working on l20n files, the editing space will look a bit different.
 
 ![Translation workspace for l20n files](/assets/images/pontoon/ui/workspace_l20n.png)
 
-In this example, the string has 2 attributes: `value` and `title`. Both are displayed in the source section (highlighted in red), and available as separate input fields in the editing area (highlighted in orange).
+When working on l20n files, the string has 2 attributes: `value` and `title`. Both are displayed in the source section (highlighted in red), and available as separate input fields in the editing area (highlighted in orange).
```

Comment:
Not always l20n files have 2 attributes. They might have one, or more. 
The same sentence is also already there before the image.

The rest looks good.